build/template_kql/common/extend_resource.kql

```diff
@@ -82,6 +101,7 @@ resources
     type contains 'storageaccounts', tostring(replace('-', '_', tostring(iff(type =~ "microsoft.storage/storageaccounts", sku.name, properties.accountType)))),
     type == "microsoft.network/azurefirewalls", tostring(properties.sku.name),
     type == "microsoft.network/frontdoors", "Not Applicable",
+    type == "microsoft.cdn/profiles", "Not Applicable",
```

Comment:
Can we display SKU name here? => "sku.name"

Reply:
Thank you, fixed it! For further detail, please see following new commit.
https://github.com/tsubasaxZZZ/reliability-workbook/commit/5ff71d2bd19233d367871dc4ad80a2607952cf63